lib/cb/clients/base.rb

```diff
@@ -14,6 +14,14 @@ class Base
       def self.cb_client
         @cb_client ||= Cb::Utils::Api.instance
       end
+
+      def self.headers(args)
+        {
+            'Accept' => 'application/json',
+            'Authorization' => "Bearer #{ args[:oauth_token] }",
```

Comment:
Will all of the APIs that use this base client be auth code flow APIs that require a bearer token? If not, should this be changed to be more generic?

I know that the client credential APIs were previously done in a way that we called `.get` on the token object. Will that continue or be changed as well?